src/createServer.js

```diff
@@ -1,8 +1,57 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
 function createServer() {
   /* Write your code here */
   // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const normUrl = new url.URL(req.url, `http://${req.headers.host}`);
+    const path = normUrl.pathname;
+    const file = path.replace('/file', '') || 'index.html';
+
+    if (path.includes('//')) {
+      res.statusCode = 404;
+      res.setHeader('Content-Type', 'text/plain');
+      res.end('Valid request');
```

Comment:
The message 'Valid request' is misleading for a 404 status code. Consider changing it to something like 'Invalid request' or 'Malformed URL'.